pkg/questionModel/multiSelectList.go

```diff
@@ -58,6 +60,7 @@ func (m *MultiSelectList) InitializeModel(input *QuestionInput) {
 	m.header = header
 	m.itemMap = itemMap
 	m.question = input.QuestionString
+	m.errorMsg = "Atleast one option must be chosen!"
```

Comment:
Prefer to rewrite this using active voice:

```suggestion
	m.errorMsg = "Please choose at least one option"
```